### PR TITLE
Ex. 3 Get All People for Type

### PR DESCRIPTION
**Pull request by ttewah**

### CODE REVIEW

File: Assignments/stu5/api.js [hunk 1]
```diff
@@ -34,17 +34,23 @@ app.get('/ex1/persons/', cors(corsOptions), async (req, res) => {
...
+    // console.log(result)
      res.send(result);
 });
 
 
 
 //
 // GET /ex3/persons?personType={Manager|Cashier|Stock%20Person}
-//
+app.get('/ex3/persons/personType=Stock%20Person', cors(corsOptions), async (req, res) => {
```
Comment (mburolla) [inline Assignments/stu5/api.js]: This endpoint should be: `/ex3/persons/` remove the personType.

File: Assignments/stu5/data-access.js [hunk 2]
```diff
@@ -31,3 +43,15 @@ exports.getPerson = async () => {
     }
     return retval;
   };
+
+  exports.getStockPersons = async () => {
+    let personType = 'Stock Person'
```
Comment (mburolla) [inline Assignments/stu5/data-access.js]: we can't hardcode stock person.  this needs to work for all person types.

File: Assignments/stu5/test-data-access.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ const main = async () => {
     // }
     // let personId = 1
 
-    let r = await dataAccess.getPerson();
+    let r = await dataAccess.getStockPersons();
```
Comment (mburolla) [inline Assignments/stu5/test-data-access.js]: Should change the name for this to `getPersonsForType()`

Comment (mburolla) [reply Assignments/stu5/test-data-access.js]: Did this get updated?

Comment (ttewah) [reply Assignments/stu5/test-data-access.js]: I just did

**Pull request merged**